gtsam/geometry/Rot3.h

```diff
@@ -479,6 +479,12 @@ namespace gtsam {
      */
     Rot3 slerp(double t, const Rot3& other) const;
 
+    /**
+     * @brief Compute the angle (in degrees) between *this and other
```

Comment:
We did in degrees in python but I propose as part of Rot3 it should be radians. Also, we might as well make this axis and angle? Return `pair<Unit3,double>` pair? Call it `axisAngle`?

PS, naming: GTSAM has consistently used camelCase for functions, not snake_case. We should stick to that knowing it departs from Google style guide.

Reply:
I thought I followed camel case here? It's just single word and it is an object method (in contrast to a static method). Am I missing something?

I'll update the signature.

Reply:
Nothing wrong :-) It was just an FYI, should be 'axisAngle' if we do both.